### PR TITLE
BLOG-106 | Write an updated blog post how to develop angular 2 + docker with angular-boilerplate v0.2.0 release

### PR DESCRIPTION
**Pull request by hoatle**
ref: https://issues.teracy.org/browse/BLOG-106

### CODE REVIEW

File: source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown [hunk 1]
```diff
@@ -20,6 +20,14 @@ published: true
 ---
 {% img center /images/2016/09/22/angular-boilerplate.png angular-boilerplate %}
 
+
+**Note:**
+
+This post is outdated, please follow [How to develop Angular 2+ applications easily with Docker, angular-cli and angular-boilerplate](/2017/03/22/how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate/)
```
Comment (hoavt) [inline source/_posts/2016-09-22-how-to-develop-angular-2-applications-easily-with-docker-and-angular-cli.markdown]: please follow the

File: source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+All the instruction should be updated in the README file, please follow there to apply for your
+project: https://github.com/acme101/angular-hello-world/blob/develop/README.md
+
+
+Summary
+-------
+
+All the heavy lifting and best practices are documented into the corresponding projects, that's how
+we should do so that everyone can follow easily and we can support each other with ease, too.
```
Comment (hoavt) [inline source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown]: support with each other easily, too.

File: source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+Hopefully, this will help you a lot with your Angular projects and any other projects that you can
+apply similarly.
+
+Happy hacking and don't forget to let us know your feedbacks and questions by commenting!
```
Comment (hoavt) [inline source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown]: by leaving your comments.

File: source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+
+<!-- more -->
+
+
+Received Feedbacks
+------------------
+
+We received some outstanding feedbacks and questions about:
+
+- Problems when running Docker images after pulling the built Docker images
```
Comment (hoavt) [inline source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown]: Should have the full stop at these sentences

File: source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+Re-using and scaling boots productivity and cost savings.
+
+
+Best practices
+--------------
+
+By looking into the `angular-boilerplate` or `angular-hello-world`, you can see the following best
+practices:
+
+- we should build the production Docker image for production deployment
```
Comment (hoavt) [inline source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown]: - Should change "we" --> "We"
- Should have the full stop at these sentences

File: source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown (hunk 2 above)
Comment (hoatle) [inline source/_posts/2017-03-22-how-to-develop-angular-2-applications-easily-with-docker-angular-cli-and-angular-boilerplate.markdown]: I keep this on purpose

**Comment by hoatle** [comment]
@hoavt feedbacks applied.

**Pull request merged**